.devcontainer/docker-compose.workspace.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.9'
+
+networks:
+  local_network:
+    name: local_network
+
+services:
+  workspace:
+    build: ./server
```

Comment:
Since you're not doing anything special in the docker file i think you can just use`image: python:3-alpine3.19` and remove the `build:`

Reply:
done